.travis.yml

```diff
@@ -9,24 +9,27 @@ script:
   - ruby script/jack_hammer -t 2000
 matrix:
   include:
-    - rvm: 2.1.8
+    - rvm: 2.2.5
       gemfile: spec/support/gemfiles/Gemfile.rails-3.2.x
       env: DB=mysql
-    - rvm: 2.1.8
+    - rvm: 2.1.9
```

Comment:
Shouldn't this be [`Ruby 2.1.10`](https://www.ruby-lang.org/en/news/2016/04/01/ruby-2-1-10-released/)?